### PR TITLE
Create npx install tool

### PR DESCRIPTION
**Pull request by mskorsur**
Add bin folder and script to be executed via running npx. The idea is to improve developer experience and allow them to initiate content element repos by running a single command. 
The script runs 3 steps:
1. clones the repo using degit
2. install dependencies
3. run setup script (enquirer prompt)

To test it locally:

1. Pull the branch and run `npm i -g`, which installs the tailor-content-element package globally on your machine using the local code. 
2. Create the folder for the content element and position yourself inside of it. 
3. Run `npx tailor-content-element` and follow the steps.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,6 +26,9 @@
     "lint:css": "stylelint \"src/**/*.svelte\" \"src/**/*.scss\" \"src/**/*.css\"",
     "lint": "npm run lint:js && npm run lint:css"
   },
+  "bin": {
```
Comment (underscope) [inline package.json:27]: We can publish it under @tailor-cms/tce-svelte, please update name acrodingly 😉

File: bin/index.cjs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+const SUCCESS_CODE = 0;
+const ERROR_CODE = 1;
+
+function exitOnError(errorMessage) {
+  shell.echo(prettifyErrorMessage(errorMessage));
+  shell.exit(ERROR_CODE);
+}
+
+function prettifyStepTitle(title) {
+  return chalk.bgBlue.bold(title);
```
Comment (underscope) [inline bin/index.cjs]: Consider making status messages a bit less prominent 💅😉
image

File: bin/index.cjs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+function prettifyErrorMessage(errorMessage) {
+  return chalk.bgRed.bold(errorMessage);
+}
+
+
+if (!shell.which('git')) {
+  exitOnError('Sorry, this script requires git');
+}
+
+if (!shell.which('degit')) {
```
Comment (underscope) [inline bin/index.cjs]: To avoid this req we can do something like `git clone --depth 1` and delete `.git` folder (one less thing to install)

**Review by underscope — COMMENTED** [review]
🎖🎖🎖

File: package.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "tailor-content-element",
+  "name": "@tailor-cms/tce-svelte",
```
Comment (underscope) [inline package.json:2]: Need to omit private: true below ;)

File: bin/index.cjs [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  exitOnError('Sorry, this script requires git');
+}
+
+if (!shell.which('node') || !shell.which('npm')) {
+  exitOnError('Sorry, this script requires node & npm');
+}
+
+shell.echo(prettifyStepTitle('1/3 Cloning respository'));
+let cloneCommand = '';
+if (!shell.which('degit')) {
```
Comment (underscope) [inline bin/index.cjs]: What benefit do we keep if we have both? 🤔

File: bin/index.cjs [hunk 6]
```diff
@@ -0,0 +1,53 @@
+#! /usr/bin/env node
+const chalk = require('chalk');
```
Comment (underscope) [inline bin/index.cjs:2]: Breaks due to dependencies not being installed prior to this point 😉

Comment (mskorsur) [reply bin/index.cjs:2]: Seems like dependencies used by the script have to be specified as regular deps, and not the dev ones.

File: bin/index.cjs (hunk 5 above)
Comment (mskorsur) [reply bin/index.cjs]: Not much, just that if someone has degit installed already, they could[ leverage it](https://github.com/Rich-Harris/degit#wait-isnt-this-just-git-clone---depth-1) to slightly faster download the repo. But, for the sake of simplicity, I'll leave just the git clone version

File: bin/index.cjs [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+const setupSnippet = new Snippet({
+  name: 'package.json',
+  message: 'Fill out the fields in package.json',
+  required: true,
+  template: `{
+  "description": "\${description}",
+  "version": "\${version:0.0.1}",
+  "author": "\${author_name} <\${author_email}> (https://github.com/\${username})"
+}
+`
```
Comment (underscope) [inline bin/index.cjs:35]: @mskorsur Is the indentation here intentional?

File: bin/index.cjs [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+
+function prettifyStepTitle(title) {
+  return chalk.blue.bold(title);
+}
+
+function prettifyErrorMessage(errorMessage) {
+  return chalk.bgRed.bold(errorMessage);
+}
+
+async function updatePackageJson(updates) {
```
Comment (underscope) [inline bin/index.cjs]: Nitpick; consider renaming updates => data

File: bin/index.cjs [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+  });
+  const deleteGitCommand = shell.exec('rm -rf .git');
+  const deleteBinCommand = shell.exec('rm -rf bin');
+
+  if (deleteGitCommand.code !== SUCCESS_CODE || deleteBinCommand.code !== SUCCESS_CODE) {
+    exitOnError('Cleanup failed');
+  }
+}
+
+function displayInstructions() {
```
Comment (underscope) [inline bin/index.cjs:132]: Consider having it as one of the steps 💅

**Review by underscope — COMMENTED** [review]
🏆

File: bin/index.cjs (hunk 7 above)
Comment (mskorsur) [reply bin/index.cjs:35]: Yes, it looks weird in the console if it's formatted properly in the code.

**Pull request merged**